features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -492,10 +516,6 @@ int lwip_bringdown(void)
         netif_set_down(&lwip_netif);
     }
 #endif
-
```

Comment:
It wasn't there on the previous commit either but it's on master so I'm guessing it got lost during one of the merges somewhere. I'll go through full diff between our merge and trunk tomorrow to see if there's anything else missing.